### DISCUSSION TITLE
Question: SFTPGO for encrypting external SFTP backend?

### DISCUSSION THREAD

**Discussion by Antergosgeek** [drakkan/sftpgo#1456] [Q&A] [answered]
Hi,

I have just started using SFTPGO and so far I love it. Now I see you can encrypt local storage, but wanted to check if it is also possible to use SFTPGO to encrypt an external SFTP backend. 

I would like to use it as my one go to for something like this. I am running a server at home on which I have installed and configured SFTPGO but I also have external SFTP storage that I would like to use. However I typically encrypt external stored files. So far I am using Rclone for that but would be nice to have a one stop solution.

**Comment by drakkan** [comment, marked as answer]
Hello, this feature is not currently supported, sorry

**Reply by Antergosgeek** [reply]
Not a problem, would have been great but at least now I know. Still, awesome project!

**Comment by drakkan** [comment]
> Not a problem, would have been great but at least now I know. Still, awesome project!

Thank you for the kind words, it may be added in the future, depends on my free time and/or a company interested in sponsoring the implementation and maintenance of this feature.
